### ISSUE TITLE
<Response [200]>

### ISSUE THREAD

**Issue by lol-linux** [makiisthenes/TiktokAutoUploader#97]
C:\Users\coucou\Downloads\TiktokAutoUploader-main\TiktokAutoUploader-main>python cli.py upload -u a -v "a.mp4" -t "Titre de ma vidéo"
User successfully logged in.
Tiktok Datacenter Assigned: useast2a
Uploading video...
[-] Publish failed to Tiktok, trying again...
[-] An error occured while reaching https://www.tiktok.com/tiktok/web/project/post/v1/
<Response [200]>
b'{"log_pb":{"impr_id":"202412211829274E365ABB5BDC84C1EA1D"},"status_code":5,"status_msg":"Invalid parameters"}'

**Comment by JWPapi** [comment]
I have the same issue

**Comment by xxbadgame** [comment]
Same

**Comment by notsungod** [comment]
Merry Christmas!
Go into this file: `tiktok_uploader/tiktok.py`
Go to row 175 and comment out data segment and replace it with:
`    data = {
        "post_common_info": {
            "creation_id": creation_id,
            "enter_post_page_from": 1,
            "post_type": 3,
        },
        "feature_common_info_list": [
            {
                "geofencing_regions": [],
                "playlist_name": "",
                "playlist_id": "",
                "tcm_params": '{"commerce_toggle_info":{}}',
                "sound_exemption": 0,
                "anchors": [],
                "vedit_common_info": {"draft": "", "video_id": video_id},
                "privacy_setting_info": {
                    "visibility_type": 0,
                    "allow_duet": 1,
                    "allow_stitch": 1,
                    "allow_comment": 1,
                },
                "content_check_id": "",
            }
        ],
        "single_post_req_list": [
            {
                "batch_index": 0,
                "video_id": video_id,
                "is_long_video": 0,
                "single_post_feature_info": {
                    "text": title,
                    "text_extra": text_extra,
                    "markup_text": markup_text,
                    "music_info": {},
                    "poster_delay": 0,
                    "cloud_edit_video_height": 2160,
                    "cloud_edit_video_width": 1920,
                    "cloud_edit_is_use_video_canvas": False,
                },
            }
        ],
    }
`

**Comment by simix** [comment]
the issue is with : "schedule_time": schedule_time + int(time.time()),

**Comment by makiisthenes** [comment]
These changes have been added, including the schedule time field.

**Comment by simix** [comment]
> These changes have been added, including the schedule time field.

I know, I mean it's still getting <Response [200]>
But when I comment schedule time it's Works

**Comment by makiisthenes** [comment]
`schedule_time` field might be added in a different part of the payload.

**Comment by karta808** [comment]
I'm running into a similar issue, any ideas?

`C:\Users\Kevin\Development\TikTikAutoUpload\TiktokAutoUploader>python cli.py upload --user 2 -v "C:\Users\Kevin\Dropbox\Apps\TikTok_Download\TikTok_Content\test.MOV" -t "test"
User successfully logged in.
Tiktok Datacenter Assigned: useast2a
Uploading video...
[-] Publish failed to Tiktok, trying again...
[-] An error occured while reaching https://www.tiktok.com/tiktok/web/project/post/v1/
<Response [200]>
b'{"log_pb":{"impr_id":"202412271845229C745D10DE092F43B562"},"status_code":5,"status_msg":"Invalid parameters"}'

C:\Users\Kevin\Development\TikTikAutoUpload\TiktokAutoUploader>`

**Comment by alawi2306** [comment]
I fixed this by scheduling the video, add the -sc flag plus the amount of seconds you want to add e.g -sc 200. 

We can schedule the video 10 seconds ahead (effectively uploading it immediately)

# Upload from videos path
python cli.py upload --user my_saved_username -v "video.mp4" -t "My video title" -sc 200

**Comment by awoods8005** [comment]
"python cli.py upload --user my_saved_username -v "video.mp4" -t "My video title" -sc 200"

I tried this but got an error saying "Cannot schedule video in more than 10 days or less than 20 minutes"

So now it only works if I include the -sc but have more than 1200 seconds lmao

**Comment by alawi2306** [comment]
Yes apologies for that. The cli command must be more than 1200 seconds
